.travis.yml

```diff
@@ -11,14 +15,33 @@ env:
     - "TEST_SUITE=unittests"
     - "TEST_SUITE=integrationtests"
     - "TEST_SUITE=integrationtestsdev"
+
+dist: xenial
+
+sudo: required
+
 addons:
   apt:
     packages:
       - chromium-chromedriver
       - xvfb
+  mariadb: '10.0'
+
 before_script:
+  - java -version
+  - echo $JAVA_HOME
+  - sudo add-apt-repository ppa:openjdk-r/ppa -y
+  - sudo apt-get update
+  - sudo apt-get install openjdk-8-jdk -y
+  - export JAVA_HOME="/usr/lib/jvm/java-8-openjdk-amd64"
+  - export PATH=${JAVA_HOME}/bin:$PATH
+  - java -version
+  - echo $JAVA_HOME
```

Comment:
A bit of a workaround while IRIDA still depends on java 8 which is not negatively supported by travic-ci xenial